Task1.Logger/TrackingEntityAttribute.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace Task1.Logger
+{
+    public class TrackingEntityAttribute : Attribute { }
```

Comment:
It is a bit confusing, there are duplicates of the classes. 

> Place the generated types in a separate class library. In a console application, connect this class
> library and test the work of types from it.

The attributes and logger should be in a separate class library .
Console application should reference them and run logger.

Reply:
My Task contains 2 separate projects:
1.LoggerBL (Logger class)
2.TestClassLibrary (Test class)